packages/react-select/src/components/Menu.js

```diff
@@ -370,9 +372,17 @@ export const menuListCSS = ({
   WebkitOverflowScrolling: 'touch',
 });
 export const MenuList = (props: MenuListComponentProps) => {
-  const { children, className, cx, getStyles, isMulti, innerRef, innerProps } = props;
+  const { children, className, cx, getStyles, isMulti, innerRef, innerProps, id } = props;
+
+  // aria attributes makes the JSX "noisy", separated for clarity
+  const ariaAttributes = {
+    role: 'listbox',
```

Comment:
Oh I missed this.

How does this work in conjunction with the changes made [here](https://github.com/JedWatson/react-select/pull/4322/files#diff-28278ba3fce9a624f09452aa1d9ee109d9e183bce08a032eff72a83a03a9252a) is role defined twice?

Reply:
The innerProps of a custom component always override the the existing props including aria attributes which is the basis of this solution.